SCClassLibrary/Common/Collections/SequenceableCollection.sc

```diff
@@ -350,10 +350,12 @@ SequenceableCollection : Collection {
 		^this.species.fill(newSize, { |i| this.blendAt(i * factor) })
 	}
 
-	remove { arg item;
+	removeAt { |index| ^this.subclassResponsibility(thisMethod) }
+	remove { |item|
 		var index = this.indexOf(item);
```

Comment:
the inner caret (^) should be removed

Reply:
I've done this, but there is loads of code that already does this.